.github/workflows/e2e-node.yml

```diff
@@ -7,26 +7,31 @@ on:
 
 env:
   CI: true
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 jobs:
   e2e-node:
     # Running end-to-end tests requires accessing secrets which aren't available to dependabot.
     if: github.actor != 'dependabot[bot]'
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     environment:
       name: ${{ matrix.environment-name }}
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [16.x]
-        # Note: "Inrupt Production" and "Inrupt 1.1" have been disabled due to the migration
-        # "Inrupt Dev-Next" is a 2.x environment.
-        environment-name: ["ESS Dev-Next", "NSS"]
+        node-version: [16.x, 14.x]
+        environment-name: ["ESS PodSpaces", "NSS"]
```

Comment:
```suggestion
#        environment-name: ["ESS PodSpaces", "NSS"]
        environment-name: ["ESS PodSpaces"]
```

Let's de-provision NSS for now, and focus on getting ESS PodSpaces and ESS Dev-Next rolled out as testing environments. We can add this back in across all the SDKs in the future, if we decide to continue testing against NSS directly from here — we need to actually come up with our formal strategy for testing other servers than ESS.

Reply:
I agree that we need an overarching strategy, but why remove the NSS tests that are currently implemented and working?

Reply:
Because the NSS environment is not managed by our new e2e-environments.

Reply:
Though it is incidentally supported here: https://github.com/inrupt/typescript-sdk-tools/blob/main/packages/internal-test-env/index.ts#L34-L39

Reply:
Let's get a plan in place for rolling out e2e-environments for NSS and CSS in general, then we can add it back in.

Reply:
My concern is that we already have a lot of things to get done before testing on CSS/NSS, so if we disable this now we'll be blind for a while, even when we roll out stuff like the authn refactor, which I'd prefer to be aware of it it breaks NSS compatibility.